### ISSUE TITLE
In Outlook 2013, NVDA does not read the list of categories found within the Options dialog.

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#3488]
**Reported by sumandogra on 2013-09-02 13:45**
Outlook 2013

The category type is not announced in Outlook options dialog and a TAB after cancel button is silent and one does not come to know about the category name. only after going to the group one can identify the type of category.
Steps to reproduce:
1.  Open options dialog from file menu.
2.  TAB to cancel button.
3.  The Cancel button is announced.
4.  Again TAB.
5.  Nothing is announced.
6.  Again press TAB and one lands in a group of a type of categorised controls to help customise Outlook.
7.

**Comment by ehollig** [comment]
This works correctly for me in Outlook 2016 and @josephsl reported it worked in 2010 and 2013 as well. Seeing as the original poster did not report with results with 2014.2 (or newer) and we are unable to contact the original author for additional information, closing
